wx-core/src/lib.rs

```diff
@@ -718,10 +797,6 @@ impl<'a, P: WindowMethods> ComboBoxBuilder<'a, P> {
         self.id = id;
         self
     }
-    pub fn value(&mut self, value: &str) -> &mut Self {
-        self.value = value.to_string();
-        self
-    }
```

Comment:
Why adding support to wxFilePickerCtrl removes other class(here, ComboBoxCtrl)'s builder method?

Is this mistake? If so, please fix this.